app/views/compute_resources/show/_vmware.html.erb

```diff
@@ -6,3 +6,7 @@
   <td><%= _("Console passwords") %></td>
   <td><%= @compute_resource.set_console_password? ? _("Enabled") : _("Disabled") %></td>
 </tr>
+<tr>
+  <td><%= _("Caching") %></td>
+  <td><%= @compute_resource.caching_enabled ? _("Enabled") : _("Disabled") %></td>
```

Comment:
I actually do prefer the text vs. the icons to be consistent in the UI. We can change this to icons in a separate PR. ´pficon-warning-triangle-o´ is too much warning. If a user disables the caching, it won't break anything.